products/samsungmobile.md

```diff
@@ -61,18 +251,48 @@ releases:
     eol: false
     releaseDate: 2021-08-27
 
+-   releaseCycle: "Galaxy Z Fold3 5G"
+    support: true
+    eol: false # https://doc.samsungmobile.com/SM-F926B/SER/doc.html
+    releaseDate: 2021-08-27
+
+-   releaseCycle: "Galaxy Z Flip3 5G"
+    support: true
+    eol: false # https://doc.samsungmobile.com/SM-F711B/SER/doc.html
+    releaseDate: 2021-08-27
+
+-   releaseCycle: "Galaxy Watch4 Classic"
+    support: true
+    eol: false # https://doc.samsungmobile.com/SM-R890/XAA/doc.html
+    releaseDate: 2021-08-27
+
+-   releaseCycle: "Galaxy Watch4"
+    support: true
+    eol: false # https://doc.samsungmobile.com/SM-R870/XAA/doc.html
+    releaseDate: 2021-08-27
+
 -   releaseCycle: "Galaxy A03s"
     support: true
     eol: false
     releaseDate: 2021-08-18
 
+-   releaseCycle: "Galaxy A12 (India)"
+    support: true
+    eol: false # https://doc.samsungmobile.com/SM-A127F/ATO/doc.html
+    releaseDate: 2021-08-12
+
 -   releaseCycle: "Galaxy Z Flip 3"
     support: true
     eol: false
     releaseDate: 2021-08-11
 
+-   releaseCycle: "Galaxy A12 Nacho"
+    support: true
+    eol: false # https://doc.samsungmobile.com/SM-A127F/ITV/doc.html
+    releaseDate: 2021-08-09
+
 -   releaseCycle: "Galaxy M21 2021"
-    support: false
+    support: 2022-10-01 # Approximate from here: https://9to5google.com/2022/10/04/samsung-android-12-update-rollout/
```

Comment:
According to https://www.gsmarena.com/samsung_galaxy_m21_2021-11015.php this one is still on Android 11 with no possibilities to upgrade to Android 12 or 13. So LGTM, but maybe we should update the comment and better document the rule ?